src/systems/physics/Physics.cc

```diff
@@ -1362,11 +1469,14 @@ void PhysicsPrivate::UpdatePhysics(EntityComponentManager &_ecm)
         if (linkEntityIt == this->linkEntityMap.end())
           return true;
 
-        auto canonicalPoseComp =
-            _ecm.Component<components::Pose>(linkEntityIt->second);
+        // set world pose of canonical link in freegroup
```

Comment:
Here's an issue when setting the pose of a nested model. Maybe we should allow `WorldPoseCmd` only on top level models? This probably applies to other `*Cmd` components as well.
![Peek 2020-08-12 00-01](https://user-images.githubusercontent.com/206116/89977084-e1e9f000-dc2f-11ea-948e-cd55c04ac5e2.gif)

Reply:
ok I  disabled gui for setting pose cmds in 6db69f494d315b09d26eabdf9edf0ca314c4d4c8 and prevented setting pose and vel commands in physics system in 451959056358b2a92041ca5ee2a448b21bc35d33